readme.md

```diff
@@ -543,6 +543,7 @@
 | [Halfmoon UI](https://www.gethalfmoon.com/)| A responsive and lightweight framework, designed for quickly building beautiful dashboards and product pages. |
 | [Metro 4](https://metroui.org.ua/index.html)| Create your site quickly and effectively with Metro 4. impressive components library built on html, css, javascript. |
 | [css-doodle](https://css-doodle.com/) | A web component for drawing patterns with CSS |
+| [CSS Gradient](https://cssgradient.io/gradient-backgrounds/) | CSS Gradient is a little website and free tool that lets you create a gradient background for websites |
```

Comment:
you can add cssgradient as its own resource with the link `https://cssgradient.io/` 
```suggestion
| [CSS Gradient](https://cssgradient.io) | CSS Gradient is a little website and free tool that lets you create a gradient background for websites |
```